### PR TITLE
Update `README` and `metadata.json` for `7.1.0-rc.1` release

### PR DESCRIPTION
**Pull request by daxian-dbw**
# PR Summary

Update `README` and `metadata.json` for `7.1.0-rc.1` release.

### CODE REVIEW

File: tools/metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "StableReleaseTag": "v7.0.3",
-  "PreviewReleaseTag": "v7.1.0-preview.7",
+  "PreviewReleaseTag": "v7.1.0-rc.1",
   "ServicingReleaseTag": "v6.2.7",
```
Comment (daxian-dbw) [inline tools/metadata.json:4]: @TravisEz13 Is this the right change for `metadata.json` for a RC release?

**Comment by rjmholt** [comment]
This looks good. Will approve when the release is out and the markdown tests pass

File: tools/metadata.json (hunk 1 above)
Comment (TravisEz13) [reply tools/metadata.json:4]: preview should stay preview

File: tools/metadata.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "StableReleaseTag": "v7.0.3",
-  "PreviewReleaseTag": "v7.1.0-preview.7",
+  "PreviewReleaseTag": "v7.1.0-rc.1",
```
Comment (TravisEz13) [inline tools/metadata.json:3]:
```suggestion
  "PreviewReleaseTag": "v7.1.0-preview.7",
```

**Review by TravisEz13 — DISMISSED** [review]
see comment

File: tools/metadata.json (hunk 2 above)
Comment (daxian-dbw) [reply tools/metadata.json:3]: How about the rest keys, especially `NextReleaseTag`? Should any of them be updated?

**Comment by daxian-dbw** [comment]
@TravisEz13 Your comment was addressed. Please take another look, thanks!

**Comment by TravisEz13** [comment]
Actually, I'm wrong.

**Comment by daxian-dbw** [comment]
Discussed with @TravisEz13 offline, and agreed that only the `PreviewReleaseTag` entry needs to be updated, to `v7.1.0-rc.1`.

**Pull request merged**